### DISCUSSION TITLE
tutorials/federation-typescript/07-contributing-entity

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#128] [course feedback]
# tutorials/federation-typescript/07-contributing-entity

https://www.apollographql.com/tutorials/federation-typescript/07-contributing-entity

**Comment by linomassarani** [comment]
Comming from "Growing your GraphQL API with TypeScript & Apollo Server" the mocked version will fail, because the mocked tracks doesn't have "artists: [{ id: "X" }]" field.

**Reply by lizhennessy** [reply]
Thanks for pointing this out, @linomassarani ! We'll catch this course up so that it's ready-to-go for those learnings coming directly from the lab.
